nsxt/policy_common.go

```diff
@@ -452,7 +452,6 @@ func getPolicyRulesFromSchema(d *schema.ResourceData) []model.Rule {
 		}
 
 		ruleList = append(ruleList, elem)
-		seq = seq + 1
```

Comment:
Is this related to the test removal? Or just piggybacking on the other change?

Reply:
no, unrelated, just some code cleaning